pmm/aws-staging-start.groovy

```diff
@@ -304,9 +304,20 @@ pipeline {
                         sudo yum -y update --security
                         sudo yum -y install https://repo.percona.com/yum/percona-release-0.1-7.noarch.rpm
                         sudo rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
-                        sudo yum -y install svn docker sysbench mysql57-server git php php-mysql php-pdo
+                        sudo yum -y install https://dev.mysql.com/get/mysql57-community-release-el7-11.noarch.rpm 
+                        sudo yum -y install git gcc make automake libtool openssl-devel ncurses-compat-libs
+                        sudo yum -y install svn docker git php php-mysql php-pdo mysql-community-devel mysql-community-client mysql-community-common mysql-community-server
```

Comment:
why do you need to list all mysql-* packages? As I remember they all are dependencies for server package. *devel package is beeded for compilation of sysbench. As it is recommended to use already compiled version of sysbench it is not needed

Reply:
I wanted to list all the needed packages so if the repo changes we can always get the packages we need, and sysbench is no longer available in repo in linux 2 so I had to compile it from source

Reply:
I followed the official amazon linux 2 build for that

https://aws.amazon.com/blogs/database/running-sysbench-on-rds-mysql-rds-mariadb-and-amazon-aurora-mysql-via-ssl-tls/

Reply:
I agree with Evgeniy - we don't need to compile sysbench on every run, `sudo yum -y install sysbench` should just work.

Reply:
It is not working on amazon linux 2

Reply:
Solved - no devel packaged are installed.